kubernetes/start

```diff
@@ -30,7 +30,7 @@ while (( $interactive_mode == 1)) ; do
   esac
 done
 
-./cleanup || true
+./cleanup
```

Comment:
this shouldn't be removed, it will cause the whole script to fail if docker didn't find any images/containers to remove.